starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/gradlePlugins.rocker.raw

```diff
@@ -13,7 +13,7 @@ Features features
 
 plugins {
 @if (features.contains("asciidoctor")) {
-    id("org.asciidoctor.jvm.convert") version "3.2.0"
+    id("org.asciidoctor.jvm.convert") version "3.3.0"
```

Comment:
Should this upgrade go in 2.2. or in separated PR?

Reply:
It is fine here IMO